Dockerfile

```diff
@@ -43,14 +43,6 @@ WORKDIR /srv/selection-tool
 COPY /requirements.txt /srv/selection-tool/requirements.txt
 RUN pip install -r requirements.txt
 
-#install the schematron-nokogiri gem from BuildingSync org
-WORKDIR /srv
-RUN git clone https://github.com/BuildingSync/schematron
-WORKDIR /srv/schematron
-
-RUN gem build schematron-nokogiri.gemspec
-RUN gem install schematron-nokogiri-0.0.3.gem
```

Comment:
Don't need this anymore. wasn't sure if we still needed to install `ruby` and `ruby-dev` above, but kept them anyways.

Reply:
we don't need ruby and ruby-dev anymore, no.  You can remove